server/app/controllers/users.js

```diff
@@ -0,0 +1,36 @@
+import bcrypt from 'bcryptjs';
+import jwt from 'jsonwebtoken';
+import db from '../db/dbconnect';
+
+
+/**  create a User
+ * @param {string} email
+ * @param {string} password
+ * @param {string} user
+ * @param {string} address
+ * @return {string} token
+ * @public
+*/
+const createUser = (email, password, user, address) => {
+  const hashedPassword = bcrypt.hashSync(password, 8);
+  return db.one(`INSERT INTO users (email, password, name, address) 
```

Comment:
Wouldn't it be better to abstract your database calls away into db helper functions?